src/main/java/com/project200/undabang/member/entity/PreferredExercise.java

```diff
@@ -39,6 +39,15 @@ public class PreferredExercise {
     @Builder.Default
     private LocalDateTime preferredExerciseCreatedAt = LocalDateTime.now();
 
+    @ColumnDefault("CURRENT_TIMESTAMP")
+    @Column(name = "preferred_exercise_updated_at")
+    @Builder.Default
+    private LocalDateTime preferredExerciseUpdatedAt = LocalDateTime.now();
+
+    @NotNull
+    @Column(name = "preferred_exercise_date", nullable = false, columnDefinition = "TINYINT UNSIGNED DEFAULT 0")
+    private Byte preferredExerciseDate;
```

Comment:
```suggestion
    @NotNull
    @Column(name = "preferred_exercise_date", nullable = false, columnDefinition = "TINYINT UNSIGNED DEFAULT 0")
    @Builder.Default
    private Byte preferredExerciseDate = (byte) 0;
```

그냥 보고 쓴거라 문법 확인 한 번 더 부탁드릴게요.

Reply:
해당사항 적용했습니다!